Cannot find module 'ts-toolbelt' or its corresponding type declarations

When using `mqtt-pattern` and trying to compile the code I get the following error: 

```sh
$ tsc --noEmit
node_modules/mqtt-pattern/index.d.ts:2:24 - error TS2307: Cannot find module 'ts-toolbelt' or its corresponding type declarations.

2 import type { F } from "ts-toolbelt";
                         ~~~~~~~~~~~~~


Found 1 error in node_modules/mqtt-pattern/index.d.ts:2

error Command failed with exit code 2.
```

Problem is that `ts-toolbelt` is neither a `dependency` nor a `peerDependency` so it will not be installed with this library. My current workaround is to install it myself as `devDependency`. However, it would be better if `mqtt-pattern` would handle this.

Spec:
mqtt-pattern: 2.0.0
node: v18.12.1
typescript: 4.9.3

--- a/package.json
+++ b/package.json
@@ -26,10 +26,10 @@
   "devDependencies": {
     "eslint": "^3.19.0",
     "eslint-plugin-node": "^4.2.2",
-    "tape": "^4.6.3",
-    "ts-toolbelt": "^9.6.0"
+    "tape": "^4.6.3"
   },
   "dependencies": {
-    "mqtt-match": "^1.0.2"
+    "mqtt-match": "^1.0.2",
+    "ts-toolbelt": "^9.6.0"
   }
 }

